<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Preferences</class>
 <widget class="QWidget" name="Preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_honeycomb">
          <attribute name="icon">
           <iconset resource="ui.dialogs.dialogicons.qrc">
            <normaloff>:/part/dlg_honeycomb</normaloff>:/part/dlg_honeycomb</iconset>
          </attribute>
          <attribute name="title">
           <string>Honeycomb</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::FieldsStayAtSizeHint</enum>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="dimensionsLabel1">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Default&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;dimensions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="honeycombGLayout">
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="horizontalSpacing">
                <number>-1</number>
               </property>
               <item row="0" column="0">
                <widget class="QSpinBox" name="honeycombRowsSpinBox">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="honeycombColsSpinBox">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>32</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="hRowsLabel">
                 <property name="text">
                  <string>rows</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="hColsLabel">
                 <property name="text">
                  <string>columns</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QSpinBox" name="honeycombStepsSpinBox">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="singleStep">
                  <number>1</number>
                 </property>
                 <property name="value">
                  <number>2</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="hBasesLabel">
                 <property name="text">
                  <string>bases (x21)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_square">
          <attribute name="icon">
           <iconset resource="ui.dialogs.dialogicons.qrc">
            <normaloff>:/part/dlg_square</normaloff>:/part/dlg_square</iconset>
          </attribute>
          <attribute name="title">
           <string>Square</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="dimensionsLabel2">
               <property name="text">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Default&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;dimensions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <layout class="QGridLayout" name="squareGLayout">
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="horizontalSpacing">
                <number>-1</number>
               </property>
               <item row="0" column="0">
                <widget class="QSpinBox" name="squareRowsSpinBox">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSpinBox" name="squareColsSpinBox">
                 <property name="minimum">
                  <number>6</number>
                 </property>
                 <property name="maximum">
                  <number>200</number>
                 </property>
                 <property name="value">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="sRowsLabel">
                 <property name="text">
                  <string>rows</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="sColsLabel">
                 <property name="text">
                  <string>columns</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QSpinBox" name="squareStepsSpinBox">
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="sBasesLabel">
                 <property name="text">
                  <string>bases (x32)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::FieldsStayAtSizeHint</enum>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="autoscafLabel">
           <property name="text">
            <string>Auto-scaffold style:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="autoScafComboBox">
           <item>
            <property name="text">
             <string>Mid-seam</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Raster</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="zoomSpeedLabel">
           <property name="text">
            <string>Mousewheel zoom speed:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="zoomSpeedSlider">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>50</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="defaultToolComboBox">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Select Tool</string>
            </property>
            <property name="icon">
             <iconset resource="ui.dialogs.dialogicons.qrc">
              <normaloff>:/pathtools/dlg_select</normaloff>:/pathtools/dlg_select</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pencil Tool</string>
            </property>
            <property name="icon">
             <iconset resource="ui.dialogs.dialogicons.qrc">
              <normaloff>:/pathtools/dlg_pencil</normaloff>:/pathtools/dlg_pencil</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Paint</string>
            </property>
            <property name="icon">
             <iconset resource="ui.dialogs.dialogicons.qrc">
              <normaloff>:/pathtools/dlg_paint</normaloff>:/pathtools/dlg_paint</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Add Seq</string>
            </property>
            <property name="icon">
             <iconset resource="ui.dialogs.dialogicons.qrc">
              <normaloff>:/pathtools/dlg_addseq</normaloff>:/pathtools/dlg_addseq</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="defaultToolLabel">
           <property name="text">
            <string>Default tool at startup:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::RestoreDefaults</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plugins">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="pluginTableWidget">
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Path</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPluginButton">
         <property name="text">
          <string>Add Plugin</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>zoomSpeedSlider</tabstop>
 </tabstops>
 <resources>
  <include location="ui.dialogs.dialogicons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>addPlugin()</slot>
  <slot>handleButtonClick(QAbstractButton*)</slot>
  <slot>setHoneycombRows(int)</slot>
  <slot>setHoneycombCols(int)</slot>
  <slot>setHoneycombSteps(int)</slot>
  <slot>setSquareRows(int)</slot>
  <slot>setSquareCols(int)</slot>
  <slot>setSquareSteps(int)</slot>
  <slot>setAutoScaf(int)</slot>
  <slot>setStartupTool(int)</slot>
  <slot>setZoomSpeed(int)</slot>
 </slots>
</ui>
